src/core.c/Rat.pm6

```diff
@@ -83,6 +93,7 @@ multi sub CREATE_RATIONAL_FROM_INTS(Int:D $nu, Int:D $de, Any, Any) is raw {
            nqp::p6bindattrinvres(nqp::create(Rat),Rat,'$!numerator',$nu),
            Rat,'$!denominator',$de
          )
+      !! $*RAT-UPGRADE-POLICY.UPGRADE-RAT(nu, de)
```

Comment:
Also, I'd suggest a modification to the above comment (`# do we need to downgrade to float?`) on line 81/91 since now that's not the only option.

Reply:
Yeah, fixed in the next commit  :-)